### PR TITLE
[sync] 2022/07/06

### PR DESCRIPTION
**Pull request by Vadaski**
ref #374 

新增文章
- Performance: src/_includes/docs/platform-view-perf.md
- Plugins in Flutter tests:  src/development/packages-and-plugins/plugins-in-tests.md
- AndroidX migration: src/development/platform-integration/android/androidx-migration.md
- "Binding to native Android code using dart:ffi":  src/development/platform-integration/android/c-interop.md
- Hosting native Android views in your Flutter app with Platform Views:  src/development/platform-integration/android/platform-views.md
- Adding an Apple Watch extension:  src/development/platform-integration/ios/apple-watch.md
- iOS debugging: src/development/platform-integration/ios/ios-debugging.md
- Hosting native iOS views in your Flutter app with Platform Views:  src/development/platform-integration/ios/platform-views.md
- "Binding to native macOS code using dart:ffi":  src/development/platform-integration/macos/c-interop.md
- Developer Relations Engineer, iOS Focus:  src/jobs/dre.md
- Tech Lead Manager:  src/jobs/tlm.md
- Default `PrimaryScrollController` on Desktop:  src/release/breaking-changes/primary-scroll-controller-desktop.md

### CODE REVIEW

File: src/development/platform-integration/android/plugin-api-migration.md [hunk 1]
```diff
@@ -51,7 +52,7 @@ Most of the [Flutter plugins][] provided by the flutter.dev
 team have been migrated already. (Learn how to become a
 [verified publisher][] on pub.dev!) For an example of
 a plugin that uses the new APIs, see the
-[battery package][].
+[battery plus package][].
```
Comment (chenglu) [inline src/development/platform-integration/android/plugin-api-migration.md:55]: 像这种问题就可以通过对本 PR 页面的检查来找到遗漏的地方~

Comment (chenglu) [reply src/development/platform-integration/android/plugin-api-migration.md:55]: 英文原文改动中文没有随之更改

File: src/testing/index.md [hunk 2]
```diff
@@ -88,6 +88,19 @@ or run `flutter test --help` in your terminal.
...
 你可以在终端执行 `flutter test --help` 命令获得更多有关单元测试的帮助：
 
+{{site.alert.note}}
+
+  If you're writing unit tests for code that
+  uses plugins, see
+  [Plugins in Flutter tests][].
+
+  如果你在为 plugin 编写单元测试，
+  请查看[如何测试 Flutter Plugin][Plugins in Flutter tests]。
```
Comment (AlexV525) [inline src/testing/index.md:98]:
```suggestion
  请查看 [如何测试 Flutter Plugin][Plugins in Flutter tests]。
```

File: src/development/platform-integration/ios/c-interop.md [hunk 3]
```diff
@@ -1,37 +1,56 @@
...
+and loading iOS native code within a Flutter plugin or app.
 
 您必须首先确保本地代码已加载，并且其符号对 Dart 可见，
 然后才能在库或程序使用 FFI 库绑定本地代码。
 本页主要介绍如何在 Flutter 插件或应用程序中编译、打包和加载本地代码。
 
 This tutorial demonstrates how to bundle C/C++
 sources in a Flutter plugin and bind to them using
-the Dart FFI library on both Android and iOS.
+the Dart FFI library on iOS.
```
[the remark this answers is not in the document]
Comment (chenglu) [reply src/development/platform-integration/ios/c-interop.md:56]: OK, 我来弄。

**Review by Vadaski — COMMENTED** [review]
我来处理下这两处，其他 LGTM，谢谢路哥和A少帮忙修改了不少的地方!

**Review by AlexV525 — APPROVED** [review]
RSLGTM

**Pull request merged**